include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -307,6 +307,9 @@
 #    define _ONEDPL_CPP20_RANGES_PRESENT 0
 #endif
 
+// Feature public macros
+#define ONEDPL_HAS_RANGES_API  _ONEDPL_CPP20_RANGES_PRESENT
```

Comment:
Will it be fine to recommend inclusion of both `onedpl_config` and `version` headers if #945 is implemented?